src/seedu/addressbook/data/person/Name.java

```diff
@@ -55,6 +55,7 @@ public boolean equals(Object other) {
                 && this.fullName.equals(((Name) other).fullName)); // state check
     }
 
+    // @question: why do we need hashcode for this?
```

Comment:
Elsewhere in the code there is a comment that says "use this hashCode" and reasoning. Find it 🙂 

More importantly, these kinda clarifications should be left to the comments on the VCS provider (like these comments) rather than being a part of the code. It may become irrelevant after a point and may cause confusion when it is a part of the code.